packages/service-library/src/servicelib/rabbitmq.py

```diff
@@ -152,3 +186,78 @@ async def publish(self, exchange_name: str, message: Message) -> None:
                 aio_pika.Message(message.encode()),
                 routing_key="",
             )
+
+    async def rpc_request(
+        self,
+        namespace: RPCNamespace,
+        method_name: str,
```

Comment:
can we also have a type for method_name?
and I guess there must be limitations in the names and actually also with the namespace right?

Reply:
Yes will add a type for this one.
I have added a check for the total length of namespace and method_name